storage/migrations/14_runtime_account_stats.up.sql

```diff
@@ -0,0 +1,37 @@
+BEGIN;
+
+CREATE TABLE stats.runtime_accounts
```

Comment:
not sure if it belongs in this schema but it seemed to fit better than in `analysis`

Reply:
At least so far, `stats` has been for statistics that are:
- Less time sensitive; we do not calculate them in real time
- Calculated in dedicated analyzers (largely because of the previous point). We might even want to move them out of analyzers into an off-the-shelf data analysis product in the future; it's a common problem in the industry.
- Keyed by time, and available for any point in history

So they're quite a different beast.

I'd put this into `chain.runtime_accounts`; it's just another "live" dead-reckoned value, like total_gas_used, or arguably even like the balances.

Reply:
Good points; updated!